graphite/graphite_tcp.go

```diff
@@ -32,6 +32,16 @@ func (s *TCPServer) SetLogOutput(w io.Writer) {
 	s.Logger = log.New(w, "[graphite] ", log.LstdFlags)
 }
 
+// SetDatabase sets database for all Graphite log output.
+func (s *TCPServer) SetDatabase(database string) {
+	s.Database = database
+}
+
+// Protocol returns a string version of the supported protocol.
+func (s *TCPServer) Protocol() string {
+	return "tcp"
+}
+
```

Comment:
I agree that it is an "Echo".  Maybe we should change the struct field to be `seriesWriter` instead of server.  I'm mostly looking for alignment.